pkg/cmd/release/create/create.go

```diff
@@ -164,8 +173,24 @@ func createRun(opts *CreateOptions) error {
 			return err
 		}
 
+		var generatedNotes *ReleaseNotes
 		var tagDescription string
 		var generatedChangelog string
+
+		params := map[string]interface{}{
+			"tag_name": opts.TagName,
+		}
+		if opts.Target != "" {
+			params["target_commitish"] = opts.Target
+		}
+		generatedNotes, err = generateReleaseNotes(httpClient, baseRepo, params)
+		if err != nil {
+			var httpErr api.HTTPError
+			if !(errors.As(err, &httpErr) && httpErr.StatusCode == 404) {
```

Comment:
Nit: perhaps `generateReleaseNotes` could return a special error type, e.g. `errNotImplemented` to detect the case of when the API is not available? Then the consumer could check for that more descriptive error and doesn't have to interface with HTTPError